lib/community.js

```diff
@@ -268,6 +268,13 @@ exports.all = {
         quote: 'Starting with vanilla Node.js we wanted to refactor our proxy server to be easier to use, understand, expand and maintain. My colleague and I created a rating system based on our criteria and evaluated several Node frameworks, including but not limited to Kraken.js, Express, restify, Rendr, Sails.js and of course hapi. Using Express meant bringing in a lot of third party middleware which would have required research and time. All the others were geared towards creating a RESTful backend from scratch which we weren\'t doing. hapi ended up rating the highest in our view. Just a couple days later we had dramatically reduced our lines of code and could move much quicker creating new routes and functionality.',
         person: 'Tyler Briles, JavaScript Engineer'
     },
+    'Major League Soccer': {
+        url: 'http://www.mlssoccer.com/',
+        logo: 'sites/league/themes/league/img/logo.png',
```

Comment:
The logo should be uploaded to the repo as well, same as the others in https://github.com/hapijs/hapijs.com/tree/master/public/img.